scripts/registry.sh

```diff
@@ -67,10 +67,11 @@ publish() {
     # Publish the packages to our local service.
     # without concurrency until https://github.com/Agoric/agoric-sdk/issues/8091
     yarn lerna publish --concurrency 1 prerelease --exact \
-      --dist-tag="$DISTTAG" --preid=dev"-$(git rev-parse --short=7 HEAD)" \
+      --dist-tag="$DISTTAG" --preid=dev \
       --no-push --no-git-reset --no-git-tag-version --no-verify-access --yes
 
-    versions=$("$thisdir/get-versions.sh" . | jq "$versions + .")
+    # Change any version prefices to an exact match.
+    versions=$("$thisdir/get-versions.sh" . | sed -e 's/"[~^]/"/g' | jq "$versions + .")
```

Comment:
More surgically, to strip caret prefixes of every value:

`jq '[to_entries[] | { key: .key, value: (.value | sub("^\\^"; "")) }] | from_entries'`

And a trick for merging two JSON files without embedding:

`jq --argfile versions B.json '. + $versions' A.json`